foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -219,40 +219,16 @@ Future<void> trackAction (String action, {Map<String, String> contextData});
 ```
 
 * _action_ contains the name of the action to track.
-* _contextData_ contains the context data to attach on this hit.
+* _contextData_ contains the context data to attach on the hit.
 
 **Example**
 
 ```dart
 FlutterACPCore.trackAction("action name",  data: {"key": "value"});
 ```
 {% endtab %}
-
-{% tab title="Cordova" %}
-#### Cordova
-
-### trackAction
-
-**Syntax**
-
-```jsx
-ACPCore.trackAction = function(action, contextData, success, fail);
-```
-
-* _action_ contains the name of the action to track.
-* _contextData_ contains the context data to attach on this hit.
-* _success_ callback is invoked when trackAction executes successfully.
-* _fail_ callback is invoked when trackAction fails.
-
-**Example**
-
-```jsx
-ACPCore.trackAction("cordovaAction", {"cordovaKey":"cordovaValue"}, successCallback, errorCallback);
-```
-{% endtab %}
```

Comment:
why'd we remove the cordova tab?

Reply:
Something I missed 😅 I'll add it back in.